From 734913528ec1c05b6b0b28b13f876155a7d8f52d Mon Sep 17 00:00:00 2001
From: Peter Jones <pjones@redhat.com>
Date: Tue, 13 Feb 2018 16:35:12 -0500
Subject: [PATCH 04/22] Force python to install to site-arch not site-packages

Signed-off-by: Peter Jones <pjones@redhat.com>
---
 src/python/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/python/Makefile.am b/src/python/Makefile.am
index 608b48e934e..c14aa8f8e6b 100644
--- a/src/python/Makefile.am
+++ b/src/python/Makefile.am
@@ -1,6 +1,6 @@
 # vim:noexpandtab:autoindent:tabstop=8:shiftwidth=8:filetype=make:nocindent:tw=0:
 
-pkgpythondir=$(pythondir)/libsmbios_c
+pkgpythondir=$(pyexecdir)/libsmbios_c
 
 if HAVE_PYTHON
 pkgpython_PYTHON = \
-- 
2.14.3

